### DISCUSSION TITLE
Can the .NET 7 Quic classes works well before Win11?

### DISCUSSION THREAD

**Discussion by GF-Huang** [dotnet/runtime#79979] [Q&A] [answered]
I can't find any docs describe which foundation library is used on Win7, Win8.1 and Win10, Schannel or OpenSSL?

**Comment by jkotas** [comment, marked as answer]
Quic works out of the box on Windows 11 and Windows Server 2022 only. Older OSes are missing cryptographic APIs required to support Quic. Check https://github.com/dotnet/runtime/discussions/79736 for more details.

**Reply by GF-Huang** [reply]
The docs missing about OS requirements.

https://learn.microsoft.com/en-us/dotnet/api/system.net.quic.quicconnection?view=net-7.0
https://learn.microsoft.com/en-us/dotnet/api/system.net.quic?view=net-7.0

**Reply by jkotas** [reply]
Opened #80001
